Presentation.Web/app/services/ui-customization/ui-customization-service.ts

```diff
@@ -118,6 +120,12 @@
                         bluePrint = this.loadBluePrint(module);
                         break;
                     }
+                case Models.UICustomization.CustomizableKitosModule.ItContract:
+                {
```

Comment:
hmm... no need for a switch case here.. hte code is the same and the loadBluePrint will fail if the module is unknown so just extract

```
const persisted = this.loadPersistedPreferences(module);
const bluePrint = this.loadBluePrint(module);
```
and remove the switch case + the "let" bindings in the top of the functions